lib/ice_cube/rules/yearly_rule.rb

```diff
@@ -12,7 +12,7 @@ def initialize(interval = 1, week_start = :sunday)
     end
 
     def start_of_period(time)
-      Time.new(time.year, 1, time.day, time.hour, time.min, time.sec, time.utc_offset)
+      Time.new(time.year, 1, 1, 0, 0, 0, time.utc_offset)
```

Comment:
somewhat weird that months and days are 1 indexed but hours / minutes / seconds are 0 indexed